### PR TITLE
Update docs in accordance with the new auto-complete-config behavior

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,14 +198,14 @@ class TritonPythonModel:
...
-        with `--strict-model-config=false`. Implementing this function is optional.
-        A no implementation of `auto_complete_config` will do nothing. This function
-        can be used to set `max_batch_size`, `input` and `output` properties of the
-        model using `set_max_batch_size`, `add_input`, and `add_output`.
-        These properties will allow Triton to load the model with minimal model
-        configuration in absence of a configuration file. This function returns the
-        `pb_utils.ModelConfig` object with these properties. You can use `as_dict`
+        """`auto_complete_config` is called only once when loading the model assuming
+        the server was not started with `--disable-auto-complete-config`. Implementing
+        this function is optional. A no implementation of `auto_complete_config` will
```
Comment (nv-kmcgill53) [inline README.md:203]: >A no implementation of...

Change to "No implementation of..."

File: README.md [hunk 2]
```diff
@@ -198,14 +198,14 @@ class TritonPythonModel:
...
+        do nothing. This function can be used to set `max_batch_size`, `input` and
+        `output` properties of the model using `set_max_batch_size`, `add_input`, and
+        `add_output`. These properties will allow Triton to load the model with minimal
+        model configuration in absence of a configuration file. This function returns
+        the `pb_utils.ModelConfig` object with these properties. You can use `as_dict`
```
Comment (nv-kmcgill53) [inline README.md:208]: >You can use `as_dict`...

Change to "You can use the `as_dict`..."

File: examples/auto_complete/batch_model.py [hunk 3]
```diff
@@ -40,19 +40,17 @@ class TritonPythonModel:
...
-        returned from here will be used as the final configuration for the
-        model.
+        """`auto_complete_config` is called only once when loading the model assuming
+        the server was not started with `--disable-auto-complete-config`. Implementing
+        this function is optional. A no implementation of `auto_complete_config` will
+        do nothing. This function can be used to set `max_batch_size`, `input` and
+        `output` properties of the model using `set_max_batch_size`, `add_input`, and
+        `add_output`. These properties will allow Triton to load the model with minimal
+        model configuration in absence of a configuration file. This function returns
+        the `pb_utils.ModelConfig` object with these properties. You can use `as_dict`
```
Comment (nv-kmcgill53) [inline examples/auto_complete/batch_model.py:50]: > You can use as_dict...

Change to "You can use the as_dict..."

File: examples/auto_complete/batch_model.py [hunk 4]
```diff
@@ -40,19 +40,17 @@ class TritonPythonModel:
...
-        will allow Triton to load the model with minimal model configuration
-        in absence of a configuration file. This function returns the
-        `pb_utils.ModelConfig` object with these properties. You can use
-        `as_dict` function to gain read-only access to the
-        `pb_utils.ModelConfig` object. The `pb_utils.ModelConfig` object being
-        returned from here will be used as the final configuration for the
-        model.
+        """`auto_complete_config` is called only once when loading the model assuming
+        the server was not started with `--disable-auto-complete-config`. Implementing
+        this function is optional. A no implementation of `auto_complete_config` will
```
Comment (nv-kmcgill53) [inline examples/auto_complete/batch_model.py:45]: > A no implementation of...

Change to "No implementation of..."

File: examples/auto_complete/nobatch_model.py [hunk 5]
```diff
@@ -40,19 +40,17 @@ class TritonPythonModel:
...
-        will allow Triton to load the model with minimal model configuration
-        in absence of a configuration file. This function returns the
-        `pb_utils.ModelConfig` object with these properties. You can use
-        `as_dict` function to gain read-only access to the
-        `pb_utils.ModelConfig` object. The `pb_utils.ModelConfig` object being
-        returned from here will be used as the final configuration for the
-        model.
+        """`auto_complete_config` is called only once when loading the model assuming
+        the server was not started with `--disable-auto-complete-config`. Implementing
+        this function is optional. A no implementation of `auto_complete_config` will
```
Comment (nv-kmcgill53) [inline examples/auto_complete/nobatch_model.py:45]: > A no implementation of...

Change to "No implementation of..."

File: examples/auto_complete/nobatch_model.py [hunk 6]
```diff
@@ -40,19 +40,17 @@ class TritonPythonModel:
...
+        do nothing. This function can be used to set `max_batch_size`, `input` and
+        `output` properties of the model using `set_max_batch_size`, `add_input`, and
+        `add_output`. These properties will allow Triton to load the model with minimal
+        model configuration in absence of a configuration file. This function returns
+        the `pb_utils.ModelConfig` object with these properties. You can use `as_dict`
```
Comment (nv-kmcgill53) [inline examples/auto_complete/nobatch_model.py:50]: >You can use as_dict...

Change to "You can use the as_dict..."

File: src/pb_stub.cc [hunk 7]
```diff
@@ -197,8 +197,8 @@ Stub::RunCommand()
   }
   switch (ipc_message->Command()) {
     case PYTHONSTUB_CommandType::PYTHONSTUB_AutoCompleteRequest: {
-      // Only run this case when Triton Server is started with
-      // '--strict-model-config=false'
+      // Only run this case when autom complete was requested by
```
Comment (nv-kmcgill53) [inline src/pb_stub.cc:200]: Change "autom" to "auto"

**Review by krishung5 — APPROVED** [review]

**Review by nv-kmcgill53 — APPROVED** [review]

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**